kogito-editors-js/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+cd kogito-editors-js && yarn run format && lerna run --concurrency 1 --stream precommit --since HEAD && git add .
```

Comment:
I did have issues with running this since I don't have lerna installed globally. What helped was replacing `lerna run` with `yarn lerna run`.

Reply:
@jstastny-cz thank you, I incorporated this suggestion